test/runtests.jl

```diff
@@ -389,9 +389,29 @@ end
     ast = macroexpand(@__MODULE__(), quote
         @inline function ()
         end
-    end) |> solve_from_local
+    end) |> solve_from_local!
     tp = first_scope_except_top(ast)
     scope = tp[1]
     @test isempty(scope.bounds)
     @test isempty(scope.freevars)
 end
+
```

Comment:
Thanks for adding new test sets!
Besides, I'd ask if you are using refactor tools to rename the `solve` to `solve!`?

Reply:
Well, it was just a simple search-and-replace (`%s/\<solve\>/\0!/g` in vim).  Sorry, it was a super boring technique :)

Reply:
haha, it works! thank you!